include/sound/sof/topology.h

```diff
@@ -34,6 +34,8 @@ enum sof_comp_type {
 	SOF_COMP_EQ_FIR,
 	SOF_COMP_FILEREAD,	/**< host test based file IO */
 	SOF_COMP_FILEWRITE,	/**< host test based file IO */
+	SOF_COMP_KEYWORD_DETECT,
+	SOF_COMP_VOICE_DETECT,
```

Comment:
should we move the FILEREAD/WRITE to e.g 1000 to keep a linear list for real components?